.github/workflows/changelog_reminder.yml

```diff
@@ -0,0 +1,23 @@
+on:
+  pull_request:
+    types: [opened, ready_for_review]
```

Comment:
Nice :+1: I think this is a really good combination; it avoids a new comment being posted with every commit.

Reply:
thanks